mgrs.js

```diff
@@ -286,10 +287,10 @@ function UTMtoLL(utm) {
 /**
  * Calculates the MGRS letter designator for the given latitude.
  *
- * @private
- * @param {number} lat The latitude in WGS84 to get the letter designator
+ * @private // TODO: is this actually private? If so, why is it exported?
```

Comment:
good catch! that line should be deleted.  I recently made it public (i.e. exported it), so that I could test it: https://github.com/proj4js/mgrs/blob/master/test/test.js#L123

Reply:
Well, there's public, then there's public.  If you don't think it warrants showing up on the public API I don't think there's any problem with leaving `@private` in the JSDoc.  Typescript has an `@internal` annotation that works in a similar way, that excludes it from generated typings.

Reply:
Let's keep it the way that is now then.  I'm worried about making the API more complicated without a clear use case.  If a clear use case for people using getLetterDesignator comes up independent of forward or inverse, I'll change my mind :-)

Could you replace the TODO with `// hiding from the public API to keep things simple for users` or something to that effect?